force-app/main/default/classes/SOQL.cls

```diff
@@ -39,9 +41,13 @@ public with sharing class SOQL {
      * @return A list of Contact records.
      * Update the corresponding test class and testGetContactsByFirstName method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static List<Contact> getContactsByFirstName(String firstName) {
+        List<Contact> contacts = [
+            SELECT Id, Name
+            FROM Contact
+            WHERE FirstName = :firstName];
```

Comment:
I'm only going to suggest a slight change here, but it's all down to personal preference. This is how I write mine:
```suggestion
        List<Contact> contacts = [
                SELECT 
                        Id,
                        Name
                FROM Contact
                WHERE FirstName = :firstName
        ];
```
If there are more than 3 or 4 fields I'm selecting, I will indent and format them vertically. For 1, 2, or 3, I left them stay in line with the select statement. 

You don't have to do any of this, it is personal preference. This already looks great. Just something to consider as you may write queries where you select more and more fields.